### PR TITLE
adds creating report tickets and CORS rule

### PR DESCRIPTION
**Pull request by NNKamel**

### CODE REVIEW

**Review by clarson217 — APPROVED** [review]

File: AdminToolsLogic/LogicHelper/Mapper.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+using System;
+using AdminToolsModels.LogicModels;
+using Repository.Models;
+
+namespace AdminToolsLogic.LogicHelper
+{
+    public class Mapper
```
Comment (markmoorerev) [inline AdminToolsLogic/LogicHelper/Mapper.cs:7]: comments

File: AdminToolsModels/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
```
Comment (markmoorerev) [inline AdminToolsModels/Class1.cs]: delete this file

File: AdminToolsModels/LogicModels/ReportModel.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace AdminToolsModels.LogicModels
+{
+    public class ReportModel
```
Comment (markmoorerev) [inline AdminToolsModels/LogicModels/ReportModel.cs:5]: what's this model for?

File: AdminToolsModels/LogicModels/ReportType.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+namespace AdminToolsModels.LogicModels
+{
+    public enum ReportType
```
Comment (markmoorerev) [inline AdminToolsModels/LogicModels/ReportType.cs:6]: purpose?

File: Repository/Models/ResolvedTicket.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+
+#nullable disable
+
+namespace Repository.Models
```
Comment (markmoorerev) [inline Repository/Models/ResolvedTicket.cs:6]: purpose?

File: Repository/Models/Ticket.cs [hunk 6]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+
+#nullable disable
+
+namespace Repository.Models
```
Comment (markmoorerev) [inline Repository/Models/Ticket.cs:6]: purpose?

**Review by markmoorerev — CHANGES_REQUESTED** [review]
needs comments and more comments.

File: AdminToolsLogic/LogicHelper/Mapper.cs (hunk 1 above)
Comment (NNKamel) [reply AdminToolsLogic/LogicHelper/Mapper.cs:7]: Added comments

File: AdminToolsModels/LogicModels/ReportModel.cs (hunk 3 above)
Comment (NNKamel) [reply AdminToolsModels/LogicModels/ReportModel.cs:5]: Added comments

**Pull request closed without merging**